integr8ly/installation
# Intly 404 release version fixes

## What
Remove master/latest references where possible and ensure when a release is created that specific versions are being set.

## Why

To avoid breaking changes being made in previous integreatly releases.

## Verification Steps

master:
1. Deploy this branch to an OpenShift cluster
1. Verify everything is working and is using the desired master versions of each product/operator/whatever

Release:
1. Create a new release using the release pipeline
2. Deploy the release to an OpenShift cluster
3. Verify everything is being pulled from a release tag, an explicit commit or version (Not master or latest)

## Checklist:


## Progress


##Related PRs:

https://github.com/integr8ly/tutorial-web-app-operator/pull/18
